app/controllers/auctions_controller.rb

```diff
@@ -17,6 +16,7 @@ def show
       current_user: current_user,
       bid_error: flash[:bid_error]
```

Comment:
This is a question not about this set of commits, but I have been wondering about storing messages in flash and not just dropping them down to the typical error, notice, warning. Can you explain how this is being used?